docs/rbac/example.tf

```diff
@@ -0,0 +1,44 @@
+# Create the Google Cloud terraform provider
+provider "google" {
+  project = "{{YOUR_PROJECT_ID}}"
```

Comment:
This is a lot of places in the file that the user needs to touch, I recommend a variable block before this like
```terraform
variable project {
    id = "{{YOUR_PROJECT_ID}}"
}
```
And then throughout here change `"{{YOUR_PROJECT_ID}}"` to `var.project.id`

Reply:
Nice suggestion. Though I used type "string" since this one looks like type map.

Reply:
Good point, thanks.